service/history/timerQueueProcessorBase.go

```diff
@@ -224,7 +223,7 @@ func (t *timerQueueProcessorBase) internalProcessor() error {
 
 continueProcessor:
 	for {
-		now := t.shard.GetCurrentTime(t.clusterName)
+		now := t.now()
```

Comment:
why?

Reply:
i basically change the "t.shard.GetCurrentTime(t.clusterName)" to a closure